src/Cache/BaseDataCache.php

```diff
@@ -80,7 +80,7 @@ public function __construct(Base $cache)
      * @throws InvalidArgumentException
```

Comment:
What do you mean by “setting of two `$ttl`”?

Reply:
I had specified the `$ttl` in SimplePie.php as `$this->cache_duration + time()`. This means a period of over 52 years that the cache would be valid.